src/utils/TypeGuards.ts

```diff
@@ -49,6 +49,54 @@ export class TypeGuards {
         return (node as any).getBody() != null;
     }
 
+    /**
+     * Gets if the provided value is a Node.
+     */
+    static isNode(value: unknown): value is compiler.Node {
+        if (value == null)
+            return false;
+        const kind = (value as any)["kind"];
+        if (typeof kind !== "number")
+            return false;
+        return kind !== SyntaxKind.SingleLineCommentTrivia

```

Comment:
Not necessarily true anymore. This is wrong.

Reply:
Heh. This whole method is wrong... it's checking if the passed in object is actually a ts.Node